src/utils/compareObjectsKeysLength/index.ts

```diff
@@ -0,0 +1,9 @@
+const compareObjectsKeysLength = (object1: {}, object2: {}) => {
+  if (typeof object1 !== 'object' || typeof object2 !== 'object') {
```

Comment:
I'd be okay with removing this. Our tests should be written in TS as well from now on, so if we violate this we should be warned via the compiler as well.